### ISSUE TITLE
--generate-config wont generate the config

### ISSUE THREAD

**Issue by newadventure079** [letorbi/synoindexwatcher#52]
It spits out an error message.  I'm trying to fix the error about the invalid path but I'd like to do so via the config but it wont generate one for me

```
python -m synoindexwatcher --generate-config
synoindexwatcher: error: implicit path '/volume1/music' does not exist

Please add the paths you want to watch explicitly to the command line. For example:

python -m synoindexwatcher /volume1/MyMusic /volume1/MyPhotos /volume1/MyVideos
```

The code snippet from the readme will not generate the config output you think it will put into synoindexwatcher.conf since instead of a config file, it'll be an error message
`python -m synoindexwatcher --generate-config | sudo tee /usr/local/etc/synoindexwatcher.conf`

**Comment by letorbi** [comment]
Hi newadventure079,

the recommended way for generating a config is to try the configuration first and then to generate the config-file by adding `--generate-config` to the previously tested command line. Here is an example:

First add the paths you need explicitly to the command-line and test this configuration:
```
python -m synoindexwatcher /volume1/MyMedia
```
Second, if everything is working, add `--generate-config` to generate the config-file:
```
python -m synoindexwatcher  /volume1/MyMedia --generate-config | sudo tee /usr/local/etc/synoindexwatcher.conf
```

This should give you a config-file you can then edit further.

However, the error-reporting when generating a config-file is not optimal. I'll leave this issue open until it has been improved.

Thanks for reporting :)

**Comment by letorbi** [comment]
The issue should be fixed with the new release 0.11.5. Please open e new issue if the problem still occurs.
